src/status_im/multiaccounts/login/core.cljs

```diff
@@ -321,7 +341,7 @@
              (json-rpc/call {:method     "wallet_getSavedAddresses"
                              :on-success resolve
                              :on-error   reject})))]))
-       (.then (fn [[accounts tokens custom-tokens favourites]]
+       (.then (fn [[accounts _ tokens custom-tokens favourites]]
```

Comment:
is it possible to make this function take a map for params instead? it's no so great if we need to skip a middle function param imo